pinaxcon/settings.py

```diff
@@ -15,7 +15,12 @@
     }
 }
 
-ALLOWED_HOSTS = []
+# HEROKU: Update database configuration with $DATABASE_URL.
+import dj_database_url
+db_from_env = dj_database_url.config()
+DATABASES['default'].update(db_from_env)
```

Comment:
Should this be inside an `if` or something? Otherwise it's going to try it locally too.